app/src/main/java/app/insti/api/ServiceGenerator.java

```diff
@@ -116,4 +116,4 @@ public ServiceGenerator(Context mContext) {
     public RetrofitInterface getRetrofitInterface() {
         return retrofitInterface;
     }
-}
+}
```

Comment:
Set your IDE to add newlines at the end of files automatically.

Reply:
Ohk but will it matter?

Reply:
It matters for two reasons:
1) Creates merge conflicts
2) The next time I add something below this (say) then the last line is changed unnecessary with a extra `\n`